differenceMinMaxValue.js

```diff
@@ -0,0 +1,27 @@
+function differenceMinMaxValue(array) {
+  if (array.lenght < 1) {
+    throw new Error("Array must contain at least 2 numbers");
+  }
+
+  let max = array[0];
+  let min = array[1];
```

Comment:
In should be changed at least for the case of array length = 1. **Hint:** it's better (easier) to initialize the maximum and minimum to `array[0]`.